libs/wallet/src/vega-order-transaction-dialog/vega-order-transaction-dialog.tsx

```diff
@@ -74,7 +76,15 @@ export const VegaOrderTransactionDialog = ({
       >
         {transaction.txHash && (
           <p data-testid="tx-hash" className="break-all">
-            {t(`Tx hash: ${transaction.txHash}`)}
+            Tx hash: &nbsp;
+            <a
```

Comment:
chore (blocking): can we please add a test to verify this?

Reply:
sure, pushing it now